bin/setup.sh

```diff
@@ -5,13 +5,17 @@ curl -sL https://deb.nodesource.com/setup_6.x -o nodesource_setup.sh
 sudo bash nodesource_setup.sh
 rm nodesource_setup.sh
 
+sudo apt-get install binutils libproj-dev gdal-bin
+
 sudo apt-get update
 sudo apt-get upgrade
 
 # Install python
 sudo apt-get install -y python-pip python-dev
 # Install postgis
 sudo apt-get install -y libpq-dev libcurl4-openssl-dev postgresql postgis
+###
+sudo apt-get install binutils libproj-dev gdal-bin
```

Comment:
You call this line twice, and I don't know why you have an empty commend above it... Besides that, looks good.